pkg/network/vmicontroller/status.go

```diff
@@ -51,6 +51,10 @@ func UpdateStatus(clusterConfig clusterConfigChecker, vmi *v1.VirtualMachineInst
 
 	interfaceStatuses = append(interfaceStatuses, secondaryIfaceStatuses...)
 
+	// virt-handler adds status entries for interfaces that only exist inside the guest.
+	// These interfaces were discovered by QEMU GuestAgent
```

Comment:
I would prefer this description to be more generic at this place. E.g. preserve already-discovered interfaces from the virt-handler. That should also hint about the name for such a function helper name.

Reply:
Done.